resources/yaml/notification_email_values.yaml

```diff
@@ -21,12 +21,13 @@ APPROVE_CONTENT: "We are pleased to inform you that your submission request for
 APPROVE_SECOND_CONTENT: "A member of the CRDC staff will reach out to you with further details regarding the next steps. If you have any questions, feel free to contact the Submission Helpdesk at $contactEmail"
 APPROVE_THIRD_CONTENT: "Thank you for your submission, and we look forward to supporting your study."
 CONDITIONAL_APPROVE_SUBJECT: "Submission Request Conditionally Approved - ACTION REQUIRED"
-INACTIVE_USER_SUBJECT: "Your CRDC Submission Portal account has been disabled"
-INACTIVE_USER_CONTENT: "Your CRDC Submission Portal user account has been disabled due to $inactiveDays days of inactivity. To reinstate your account please contact $officialEmail"
-INACTIVE_ADMIN_USER_SUBJECT: "CRDC Submission Portal user account has been disabled"
+INACTIVE_USER_SUBJECT: "Your CRDC Submission Portal account has been disabled due to Inactivity"
+INACTIVE_USER_CONTENT: "our access to the CRDC Submission Portal has been disabled due to $inactiveDays days of inactivity. To reinstate your account please contact $officialEmail"
```

Comment:
I think the first word in this constant is supposed to be "Your"

Reply:
updated the typo.